documentation/modules/metrics/con-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,81 @@
+
+Cruise Control metrics are available for real-time monitoring of Cruise Control operations.
+For example, you can use metrics to monitor the status of rebalancing operations that are running or provide alerts on any anomalies that are detected in its performance.
+
+For a full list of available Cruise Control metrics, which are known as _sensors_, see the {CruiseControlSensorsDocs}.
+
+== Exposing Cruise Control metrics
+
+If you want to expose metrics on Cruise Control operations, configure the `Kafka` resource xref:proc-metrics-kafka-deploy-options-{context}[to deploy Cruise Control metrics configuration].
+You can use your own configuration or an example file provided by Strimzi called `kafka-cruise-control-metrics.yaml`.
+
+You add the configuration to the `metricsConfig` of the `CruiseControl` property in the `Kafka` resource.
+The configuration enables JMX for Cruise Control and uses the {JMXExporter} to expose Cruise Control metrics through an HTTP endpoint.
+The HTTP endpoint is scraped by the Prometheus server.
+
+.Example metrics configuration for Cruise Control
+[source,yaml,subs="+quotes,attributes"]
+----
+  apiVersion: {KafkaApiVersion}
+  kind: Kafka
+  metadata:
+    name: my-cluster
+  Spec:
+    # ...
+    cruiseControl:
+      # ...
+      metricsConfig:
+        type: jmxPrometheusExporter
+        valueFrom:
+          configMapKeyRef:
+            name: cruise-control-metrics
+            key: metrics-config.yml
+  ---
+  kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: cruise-control-metrics
+    labels:
+      app: strimzi
+  data:
+    metrics-config.yml: |
+    # _metrics configuration..._
+----
+
+== Viewing Cruise Control metrics
+
+Having exposed Cruise Control metrics, you can use a monitoring tool to view the information on the data.
+Strimzi provides an xref:assembly-metrics-config-files-str[example Grafana dashboard] to provide visualizations of Cruise Control metrics.
+The dashboard is a JSON file called `strimzi-cruise-control.json`.
+The exposed metrics provide the monitoring data when you xref:proc-metrics-grafana-dashboard-str[enable the Grafana dashboard].
+
+=== Monitoring Balancedness scores
+
+Cruise Control metrics include balancedness scores.
+Balancedness is the measure of how evenly a workload is distributed in a Kafka cluster.
+
+The Cruise Control metric for balancedness score (`balancedness-score`) differs from the
+the balancedness score in the `KafkaRebalance` resource. The former is calculated using `anomaly.detection.goals`, while the latter is calculated using `default.goals`.
+
+NOTE: Refreshing a `KafkaRebalance` resource fetches the latest state of a Kafka cluster. The refresh also requests an optimization proposal from Cruise Control. Continual refreshes can impact performance monitoring in certain circumstances. For example, the KafkaRebalance `goals` might differ from the `default.goals` used by Cruise Control.
```

Comment:
Are these `default.goals` from the Kafka CR? Or is this something else?

Reply:
Kafka `default.goals` (the `default.goals` specified in the `spec.cruiseControl.config` of the Kafka resource )

Reply:
@kyguy  I've changed to _For example, the `goals` configured in `KafkaRebalance` resource might differ from the `default.goals` used by Cruise Control._  I think this is what we're trying to say? Or should the first `goals` be `default.goals` too?  Can we say how the refreshes impact this? I'm not entirely sure we're clear on what's happening here.

Reply:
Yes so Cruise Control calculates and caches optimization proposals based on the goals configured in the `default.goals` of the Kafka resource ahead of time. This cached optimization proposal will be fetched for a `KafkaRebalance` resource if :
1. The goals configured in the `goals` section of the KafkaRebalance resource match the goals configured in the `default.goals` section of the Kafka resource.
2.  No `goals`  section is specified  in the KafkaRebalance resource

Otherwise, the `KafkaRebalance` resource cannot use the cached optimzation proposal from Cruise Control and Cruise Control must generated a new optimzation  proposal on the fly based on the the goals configured in the `goals` section of the KafkaRebalance. This is where the extra performance cost comes from.

Reply:
Thanks @kyguy 
Here's the latest. Trying to keep it simple.

![image](https://user-images.githubusercontent.com/47596553/143904565-ed68b914-b489-4ba0-9ab6-c7825a307d82.png)